spec/project.schema.json

```
@@ -13,13 +13,16 @@
           "hash": {
             "type": "string"
           },
+          "cluster": {
+            "type": "number"
```

```suggestion
            "type": "integer"
```

even though js does not make this distinction we can do so in the schema